build/testing/cli.go

```diff
@@ -118,6 +119,34 @@ Run 'flipt --help' for usage.`))); err != nil {
 		}
 	}
 
+	{
+		container := container.Pipeline("flipt")
```

Comment:
Think of these as test case names. You want them to be unique.
```suggestion
		container := container.Pipeline("flipt (no config)")
```